apps/omg_watcher/test/omg_watcher/exit_processor/standard_exit_test.exs

```diff
@@ -104,6 +102,40 @@ defmodule OMG.Watcher.ExitProcessor.StandardExitTest do
   end
 
   describe "Core.create_challenge" do
+    test "returns a deposit exiting_tx as part of the challenge response",
+         %{alice: alice, processor_empty: processor} do
+      exiting_tx = TestHelper.create_recovered([], [{alice, @eth, 10}])
+      processor = processor |> start_se_from(exiting_tx, @utxo_pos_deposit)
```

Comment:
single pipe

Reply:
not in this PR. Changing this just here would make it inconsistent with the rest of the file. Changing everywhere would obscure the PR with unrelated style changes, while I want this PR to only contain the changes relevant to/required by the change it's centered around.

Among other reasons behind this approach: your ability to review.